.vscode/settings.json

```diff
@@ -1,6 +1,16 @@
 {
+  "eslint.format.enable": true,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "graphql-config.load.rootDir": "packages/apps/graph",
+  "[typescript][typescriptreact]": {
+    "editor.formatOnSave": false,
+    "editor.codeActionsOnSave": [
+      "source.formatDocument",
+      "source.fixAll.eslint"
+    ]
+  },
```

Comment:
I'm not a fan of interrupting workflow for everyone. This is something you can add yourself to your own settings.

I suppose configuring extensions like the graphql one is fine?